### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/genesis-bw#7]
As for other Genesis child themes, eg genesis-oik, improve the accessibility to meet WCAG 2.1 AA guidelines.

**Comment by bobbingwide** [comment]
There were 21 Contrast errors in genesis-bw v0.1.1; AIM score 5.3

Red `#ff4800` needs to be darker, when used on a white background. 
This color is used in the genesis-bw child theme's `style.css` several times.
It makes sense to improve the child theme's CSS rather than using Customiser > Additional CSS  

Try `#d63d00`, except where the background color is #222222.

After the above changes, 6 Color contrast errors remained; AIM score 7.7

These were from the bobbing wide logo spelt out on the neary black background.
The simplest solution is the change the styling of .bwlogo to use a white background. 

<img width="557" height="112" alt="Image" src="https://github.com/user-attachments/assets/7ddc50cd-c0ee-4357-aaf2-5fa531f51a3f" />

**Comment by bobbingwide** [comment]
We'll need to check other pages to see if this colour needs changing to `#d63d00`. 

<img width="368" height="651" alt="Image" src="https://github.com/user-attachments/assets/5c53960b-df55-4e36-af5c-1993df2423df" />
